### PR TITLE
NSFW text vision

### PR DESCRIPTION
**Pull request by navin-kumar-k**

### CODE REVIEW

File: yubiai/nlp/nsfw_text/textNsfwDetection.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    """
+    def __init__(self, task_name="text_nsfw_detection", use_gpu=False):
+        
+        self.use_gpu = use_gpu
+        self.model_folder_path, self.model_folder_name, self.model_zip_path, self.model_zip_name = set_model_info(task_name)
+        verify_model_path(self.model_folder_path, self.model_folder_name, self.model_zip_path, self.model_zip_name)
+        self.model = self.load_model()
+        self.label_fn = lambda label: self.model.task.label_dictionary.string([label + self.model.task.label_dictionary.nspecial])
+
+    def verify_model_path_ftp(self):
```
Comment (swapaj) [inline yubiai/nlp/nsfw_text/textNsfwDetection.py:38]: We do not need this function right ?

File: README.md [hunk 2]
```diff
@@ -35,7 +37,8 @@ State-of-the-art models in AI,ML,NLP & Vision for FinTech community by Yubi's Da
     * [Fintech Document Segmentation model v1](./yubiai/vision/document_segmentation/)
 * April 2023
     * [Fintech Document Segmentation model v2](./yubiai/vision/document_segmentation/)
-
+* Jun 2023
+    * [Document Image Detection](./yubiai/vision/document_image_detection/)
```
Comment (swapaj) [inline README.md:41]: Add entry for Image NSFW here .. even though both point to same link

File: yubiai/nlp/nsfw_text/textNsfwDetection.py (hunk 1 above)
Comment (navin-kumar-k) [reply yubiai/nlp/nsfw_text/textNsfwDetection.py]: Right

File: README.md (hunk 2 above)
Comment (navin-kumar-k) [reply README.md:41]: Added

**Review by swapaj — APPROVED** [review]

**Pull request merged**